blsSignatures/blsSignatures.go

```diff
@@ -102,9 +106,10 @@ func signMessage2(priv PrivateKey, message []byte, keyValidationMode bool) (Sign
 	if err != nil {
 		return nil, err
 	}
-	g1 := bls12381.NewG1()
-	result := &bls12381.PointG1{}
-	g1.MulScalar(result, pointOnCurve, priv)
+	result := new(bls12381.G1Affine)
+	fr := new(fr.Element)
+	fr.Set(priv)
+	result.ScalarMultiplication(pointOnCurve, fr.BigInt(new(big.Int)))
```

Comment:
Same thing here, I think we can just use priv instead of fr

Reply:
Done.